iib/web/models.py

```diff
@@ -895,7 +896,7 @@ def from_json(cls, kwargs, batch=None):
                 raise ValidationError(
                     f'"{key}" should be either an empty array or an array of non-empty strings'
                 )
-
+        # TODO check this out
         # Check if no bundles and `from_index is specified
         # if no bundles and no from index then an empty index will be created which is a no-op
```

Comment:
The comment doesn't make sense. For creation of empty index we still need from_index